docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+
+    command: >
+      sh -c "python manage.py wait_for_db &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000 
```

Comment:
The `runserver` command is used to start the Django development server. For production, consider using a production-ready server like Gunicorn or uWSGI.